tests/handlers/test_register.py

```diff
@@ -43,10 +43,6 @@ def setUp(self):
             self.addCleanup,
             expire_access_token=True,
         )
-        self.macaroon_generator = Mock(
```

Comment:
I'm a bit surprised we can remove these?

Reply:
Not sure of the history, but can't see any other examples of needing to mock it out.

Reply:
it seems to be a way of checking the access token is correctly passed through by `get_or_create_user`. So let me rephrase: is it necessary to remove these and weaken the `result _token` tests below?